epcis_event_hash_generator/hash_generator.py

```diff
@@ -147,6 +149,28 @@ def compute_prehash_from_file(path, enforce=None):
     else:
         logging.error("Filename '%s' ending not recognized.", path)
 
+    return compute_prehash_from_events(events)
+
+
+def compute_prehash_from_json_str(json, enforce=None):
```

Comment:
enforce is not used here?

Reply:
as method is specific to json enforce variable is not actually required so removed it